src/components/cart.lineitem.jsx

```diff
@@ -141,6 +144,25 @@ class CartLineItem extends React.Component {
     );
   }
 
+  renderConfiguration() {
+    const { item } = this.props;
+    const keys = Object.keys(item.configuration);
+    if (keys) {
+      return keys.map(key => (
+        <li className="configuration" key={key.split(configurationPrefix)[1]}>
+          <label className="option-name">
```

Comment:
Linting error:
`153:11  error  Form label must have ALL of the following types of associated control: id  jsx-a11y/label-has-for`

Need to add the following attribute to the element:
`htmlFor="option-name"`